src/csharp/Grpc.Microbenchmarks/CompletionRegistryBenchmark.cs

```diff
@@ -43,15 +43,15 @@ public void Cleanup()
         public void Run(int threadCount, int iterations, bool useSharedRegistry)
         {
             Console.WriteLine(string.Format("CompletionRegistryBenchmark: threads={0}, iterations={1}, useSharedRegistry={2}", threadCount, iterations, useSharedRegistry));
-            CompletionRegistry sharedRegistry = useSharedRegistry ? new CompletionRegistry(environment, () => BatchContextSafeHandle.Create()) : null;
+            CompletionRegistry sharedRegistry = useSharedRegistry ? new CompletionRegistry(environment, () => BatchContextSafeHandle.Create(), () => RequestCallContextSafeHandle.Create()) : null;
```

Comment:
might be forgetting here, is this creating the `CompletionRegistry` registry the old way just for comparison?

Reply:
This benchmark actually doesn't use the factories from completion registry at all, it uses a ctx object created here: https://github.com/grpc/grpc/pull/13521/files#diff-ba961c28d30e9e908894c6d111e75d4bR55

Reply:
I see. 

one remaining comment on the `SendMessageBenchmark.cs`, otherwise lgtm